plugin/storage/integration/fixtures/queries.json

```diff
@@ -376,5 +376,37 @@
       "NumTraces": 1000
     },
     "ExpectedFixtures": ["multiple1_trace", "multiple2_trace", "multiple3_trace"]
+  },
+  {
+    "Caption": "Tag regex + Operation name + max Duration",
+    "Query": {
+      "ServiceName": "query23-service",
+      "OperationName": "query23-operation",
+      "Tags": {
+        "sameplacetag1":"random\\*"
+      },
+      "StartTimeMin": "2017-01-26T15:46:31.639875Z",
+      "StartTimeMax": "2017-01-26T17:46:31.639875Z",
+      "DurationMin": 0,
+      "DurationMax": 1000,
+      "NumTraces": 1000
+    },
+    "ExpectedFixtures": ["tags_regex_trace"]
+  },
+  {
+    "Caption": "Tag regex + Operation name + max Duration - Multiple traces",
```

Comment:
> @pavolloffay - Also, in the integration test, the "Tag Name + Operation Name" query seems to match the tags_regex_trace.json as well, even though they search on a different ServiceName and tag value.

@annanay25 do you mean this one?

Reply:
No, the one above, "Tag regex + Operation name + max Duration"

It is supposed to match with just the `tags_regex_trace` but matches with both `tags_regex_trace` and `tags_opname_trace `.

Reply:
This one also looks wrong as it uses different service name https://github.com/jaegertracing/jaeger/blob/master/plugin/storage/integration/fixtures/traces/tags_opname_trace.json

Could you please debug it and or open an issue?

Reply:
How did you fix the issue?

Reply:
I had to mangle the traceID further - 
https://github.com/jaegertracing/jaeger/pull/2049/commits/0955efa38d658f508c64c41f71c5eafdbdf0d8ac#diff-1824d3ce6ce534210f34269d2370e966